src/evm/initialize-provider.ts

```diff
@@ -0,0 +1,71 @@
+import * as node from '@api3/airnode-node';
+import * as protocol from '@api3/airnode-protocol';
+import { LogsData } from '@api3/airnode-node';
+import { ethers } from 'ethers';
+import isNil from 'lodash/isNil';
+import { ChainConfig, EVMProviderState } from '../types';
+import { retryGo } from '../utils';
+
+const rrpBeaconServerAbi = new ethers.utils.Interface(protocol.RrpBeaconServerFactory.abi).format(
+  ethers.utils.FormatTypes.minimal
+);
+
+//TODO: where to get abi from?
```

Comment:
If the contract always implements these two signatures then you shouldn't need to retrieve the ABI - this fragment will be fine I think?

Reply:
Yes, I don't expect this to change but that's just human-readable ABI. Still I get the feeling that this should be coming from the protocol or services package 🤔